recipes/r-rattle/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '5.4.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rattle
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rattle_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rattle/rattle_{{ version }}.tar.gz
+  sha256: e5d7db5692821a0375818b7acc71b3c98d859a92ed001d39f6e8b890c834e2b1
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-xml
+    - r-bitops
+    - r-dplyr
+    - r-ggplot2
+    - r-magrittr
+    - r-rpart.plot
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+  run:
+    - r-base
+    - r-xml
+    - r-bitops
+    - r-dplyr
+    - r-ggplot2
+    - r-magrittr
+    - r-rpart.plot
+    - r-stringi
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('rattle')"           # [not win]
+    - "\"%R%\" -e \"library('rattle')\""  # [win]
+
+about:
+  home: https://rattle.togaware.com/
+  license: GPL-2
```

```suggestion
  license: GPL-2-or-later
```

Cran meta data says: # License: GPL (>= 2) ... which converts to https://spdx.org/licenses/GPL-2.0-or-later.html